.travis.yml

```diff
@@ -13,4 +13,6 @@ script:
   - julia -e 'Pkg.clone("https://github.com/mbauman/Tuples.jl"); Pkg.clone("https://github.com/mbauman/RangeArrays.jl")'
   - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("AxisArrays"); Pkg.test("AxisArrays"; coverage=true)'
 after_success:
-    - julia -e 'cd(Pkg.dir("AxisArrays")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'cd(Pkg.dir("AxisArrays")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'Pkg.add("Documenter")'
+  - julia -e 'cd(Pkg.dir("AxisArrays")); include(joinpath("doc", "make.jl"))'
```

Comment:
Changing it makes sense, best to stick with the Documenter defaults unless there's a reason to deviate I think.